manager/assets/modext/widgets/media/modx.browser.js

```diff
@@ -101,17 +102,18 @@ Ext.extend(MODx.browser.View,MODx.DataView,{
 
     ,run: function(p) {
         p = p || {};
-        if (p.dir) { this.dir = p.dir; }
         Ext.applyIf(p,{
             action: 'Browser/Directory/GetFiles'
-            ,dir: this.dir
+            ,dir: p.dir ?? this.dir
```

Comment:
When building the assets with `grunt build` this throws an error:
```
 [SyntaxError]: Unexpected token: operator «?»

 filename: 'widgets/media/modx.browser.js',
  line: 107,
  col: 25,
  pos: 4294
```

Reply:
I've reverted it